src/Apps/Artist/Routes/Overview/Components/ArtistCareerHighlights.tsx

```diff
@@ -96,6 +104,8 @@ export const ArtistCareerHighlightsFragmentContainer = createFragmentContainer(
         insights {
           ...ArtistCareerHighlight_insight
           kind
+          entities
```

Comment:
What does adding these do? I think the issue is that, `insights` is totally `null` initially - then it gets 'partially' created b/c a refetch happens - and since `insights` is now not `null`, the component tries to render something, which then crashes because `entities` or other parts of that component's fragment are missing.

So I think the fix is to be ultra-defensive in the component, assume the fragment might be 'missing', so even though `entities` is not-null, need to check for its presence, and so forth.

I _think_ you're doing that above - not sure this is needed?

Reply:
This comes from fragment. If I don't have them I'll have the ts error.

![image](https://github.com/user-attachments/assets/ec3f66e6-82dc-4d2b-a7a7-c843de3fe89e)

Reply:
Yea, I'm saying I don't think you need this check. The defensive check, as you've already done, should be in the child/individual component, not this list one. Let this list one have `insights` that is partial.

It's a bit of _over_ defensiveness that also invalidates the CDN-cached query that's making this issue apparent by modifying the query - so the potential for a 'false positive' is there - https://artsy.slack.com/archives/C05EEBNEF71/p1729610632196429?thread_ts=1729601303.168049&cid=C05EEBNEF71

Reply:
I think we want to keep this check here (doch 😅 ) because if we encounter a glitched insight (such as one with relay references), the child component won't render due to the safeguard. However, in this component, we would still display the title on this [line](https://github.com/artsy/force/pull/14685/files#diff-5eaef4e9cc9c1a503eb96404aba57af63430f665267b50214066bca229acefb6R70).